.github/workflows/lint.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           echo "pip install pre-commit"
           echo "cd serve/"
           echo "pre-commit install"
-          echo "pre-commit will lint your code for you, so git add and commit those new changes and this check should become green"
+          echo "git diff --name-only main | xargs pre-commit run --files"
```

Comment:
I think it was still helpful to let the users know they need to install pre-commit then git add and commit